src/index.js

```diff
@@ -363,6 +366,8 @@ async function typeImpl(element, text, {allAtOnce = false, delay} = {}) {
       // eslint-disable-next-line no-await-in-loop
       if (delay > 0) await wait(delay)
 
+      if (currentElement.disabled) return
```

Comment:
This should be a function call right?

```suggestion
      if (currentElement().disabled) return
```

Reply:
🤦‍♀️  😂  Yes, thank you!